docs/embeddable-uis/element-login.mdx

```diff
@@ -0,0 +1,230 @@
+---
+sidebar_position: 6
+title: Login Method
+---
+
+import HelpSupportTile from "@site/src/components/elements/HelpSupportTile.js";
+import ActionContainer from "@site/src/components/elements/ActionContainer.js";
+import ActionTile from "@site/src/components/elements/ActionTile.js";
+import CodeDropdown from "@site/src/components/elements/CodeDropdown.js";
+import FlexWrapper from "@site/src/components/elements/FlexWrapper.js";
+import code from "@site/src/components/elements/codeBlock.json";
+
+import userManagementElementInstances from "/static/ui-videos/embeddable-uis/user-management-element.mp4";
+
+The Login Method for Permit Elements is a simple and secure way to embed Permit Elements into your application.
+
+### Server-side Login method
+
+In the server side, We have the `loginAs` function takes two parameters.
+You need to pass in the unique `userId` you get from your **JWT** (JSON Web Token),and a `tenant_key` or `tenantId`.
+Passing in the tenant is **compulsory** when logging in a user **server-side**.
+
+```js
+permit.elements.loginAs({ userId, tenant });
+```
+:::info IMPORTANT
+The **user must belong to the tenant** that he will be logged into. If he is not, you will see a login error saying `USER_NOT_FOUND`.
+
+If the user gets logged out, he also exits the current tenant specificed in the `loginAs` method. If you want to **change tenants** for
+a user, you need to **log them out**, and **log them back in** to a different tenant.
+:::
+
+
+
+##### How to use the `loginAs` function
+
+### Initializing Permit
+
+<CodeDropdown
+    number="1"
+    title=""
+    server
+    code={[code.initialize_node, code.initialize_python, code.initialize_dotnet, code.initialize_java]}
+    language={["javascript", "python", "c", "java"]}
+    languages={["Node", "Python", "Dotnet", "Java"]}
+>
+    {/* If you are already using Permit, you can skip this step. */}
+    In order to use Permit Elements, you need to be a **user of Permit**. When starting to use Permit, you will need to
+    initialize an instance of Permit in your backend logic. This **only needs to be done once**, both to use Permit and
+    Elements.
+    Please remember to copy your `SDK Secret Key` and pass it into the initialized Permit object.
+
+</CodeDropdown>
```

Comment:
Change this to Tabs like in - `rebac-api-call.mdx`
<img width="382" alt="image" src="https://github.com/user-attachments/assets/4a6c2330-108c-4c15-a8ef-efa937011607">

Reply:
In all file :D